---
source: test_programs/src/lib.rs
expression: "&format!(\"success: {:?}\\nexit_code: {}\\n----- stdout -----\\n{}\\n----- stderr -----\\n{}\",\noutput.status.success(), output.status.code().unwrap_or(!0), stdout, stderr)"
---
success: false
exit_code: 1
----- stdout -----

----- stderr -----
error: The value `128` cannot fit into `i8` which has range `-128..=127`
  ┌─ compile_failure/signed_integer_literal_overflow/src/main.nr:2:9
  │
2 │     foo(128)
  │         ---
  │
  = Call stack:
    1. compile_failure/signed_integer_literal_overflow/src/main.nr:2:9

Aborting due to 1 previous error
